scripts/utilities.js

```diff
@@ -14,24 +14,21 @@ const logger = {
 };
 
 const spawnOrFail = (command, args, options) => {
-  options = {
-    ...options,
-    shell: true
-  };
-  const cmd = exec(command, args, options);
+  const cmd = exec(command, args, { shell: true });
```

Comment:
Why are we no longer using the passed options?

Reply:
Because there is no callers that actually set it. I repurpose that `options` parameters to control whether we want to print out the command output instead.